### PR TITLE
Exclude SLE12 platform.

### PR DESCRIPTION
**Pull request by aplanas**
Depends on crowbar/barclamp-crowbar#1175

### CODE REVIEW

File: crowbar_framework/app/models/provisioner_service.rb [hunk 1]
```diff
@@ -28,12 +28,18 @@ def role_constraints
...
+            "suse" => "12.0"
+          }
         },
         "provisioner-base" => {
           "unique" => false,
           "count" => -1,
-          "admin" => true
+          "admin" => true,
+          "exclude_platform" => {
+            "suse" => "12.0"
```
Comment (vuntz) [inline crowbar_framework/app/models/provisioner_service.rb]: This is not correct: this is a basic role that all nodes need.

**Comment by jsuchome** [comment]
No, crowbar-related barclamps are for all nodes

**Comment by aplanas** [comment]
Close because is a crowbar related barclamp.

**Comment by vuntz** [comment]
Well, the provisioner-server bit was relevant, imho.

**Comment by aplanas** [comment]
Well, this is a bit confusing.  Maybe I can create a list of roles (barclamp-*/chef/roles/*) and decide the constraints here, so I can have a better picture.

**Comment by vuntz** [comment]
@aplanas you can see all roles in /opt/dell/chef/roles on any install ;-) But yes, we should probably have a table somewhere to document this before you create all the PR.

**Pull request closed without merging**